haiku-apps/beohms/beohms-0.1~git.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Ohm's Law calculator"
+DESCRIPTION="BeOhms is a calculator that is based of of Ohm's Law. This \
+calculator solves either voltage, resistance, or current, given two of the \
+variables."
+HOMEPAGE="https://github.com/HaikuArchives/BeOhms"
+COPYRIGHT="1999 Kyle Crane"
+LICENSE="Public Domain"
+REVISION="1"
+commit="dfdb28d7282aba7d7d616460e85692e68e27a762"
+SOURCE_URI="https://github.com/HaikuArchives/BeOhms/archive/$commit.tar.gz"
```

Comment:
In the other recipes I wrote, I didn't do that, even when I could, so, for the sake of consistency, I think it would be better to leave it as it is.